test/unit/config/config.test.js

```diff
@@ -85,6 +85,55 @@ test('when loading options via constructor', async (t) => {
   })
 })
 
+test('agent control', async t => {
+  await t.test('loads defaults', () => {
+    const config = Config.initialize({})
+    assert.deepStrictEqual(config.agent_control, {
+      enabled: false,
+      health: {
+        delivery_location: 'file:///var/lib/newrelic-agent_control/fleet',
+        frequency: 5
+      }
+    })
+  })
+
+  await t.test('loads from env', () => {
```

Comment:
we have a separate file for env var conversion: `test/unit/config/config-env.test.js`